src/utils/keyboardNavigation.js

```diff
@@ -0,0 +1,151 @@
+import nodeVisitor from './nodeVisitor'
+
```

Comment:
I like how you came up with this idea of keeping keyboard navigation to this one single place - this is a great design in terms of separation of concerns. I would move it out of utils though. In general, I'm not a fan of `utils` or `common` or `shared` folders in any project. Over time, they become a dumping ground of sorts (I know we have a bunch of stuff in utils and more keep creeping up, but trust me - if I could find a better place for them, I would 😃).

Since this deals with navigating the tree, I think it should be in `tree` or `tree-manager`. Given how little it has to do with any UI elements and how much it has to do with its "utility" nature, I think `tree-manager` is a better place between the two.

The same goes for `nodeVistor` and aria helpers should go to a new folder called `a11y`. In general, anything that is more descriptive than the vague `utils` is preferable.

Thoughts?

Reply:
Agree with keyboardNavigation and nodeVisitor but hesistant about a11y. Currently only has "one helper" and things needed for axe-testing.

Reply:
I hear ya and I'll let you be the judge here. All I'm saying is to consider not having more stuff in utils ***if*** possible.

Reply:
a11y moved